### ISSUE TITLE
MacOS Sierra 10.12.6 inc_vendor.cl file not found

### ISSUE THREAD

**Issue by ccakgunduz** [hashcat/hashcat#1308]
I have installed the github version of Hashcat on my MacOs Sierra with using the instructions of [Build.md](https://github.com/hashcat/hashcat/blob/master/BUILD.md).

When I run `./hashcat --help`, it works fine. However when I try to test my graph card to with
`./hashcat --benchmark` it throws an error for the graph cards shown as below:

```
pc-me:hashcat root# ./hashcat --benchmark
hashcat (v3.6.0-265-ga85be1d0) starting in benchmark mode...

Benchmarking uses hand-optimized kernel code by default.
You can use it in your cracking session by setting the -O option.
Note: Using optimized kernel code limits the maximum supported password length.
To disable the optimized kernel code in benchmark mode, use the -w option.

OpenCL Platform #1: Apple
=========================
* Device #1: Intel(R) Core(TM) i7-6700HQ CPU @ 2.60GHz, skipped.
* Device #2: Intel(R) HD Graphics 530, 384/1536 MB allocatable, 24MCU
* Device #3: AMD Radeon Pro 450 Compute Engine, 512/2048 MB allocatable, 10MCU

Hashtype: MD4

clBuildProgram(): CL_BUILD_PROGRAM_FAILURE

<program source>:8:10: fatal error: 'inc_vendor.cl' file not found
#include "inc_vendor.cl"
         ^

* Device #2: Kernel /Users/me/GitHub/hashcat/OpenCL/m00900_a3-optimized.cl build failed - proceeding without this device.

clBuildProgram(): CL_BUILD_PROGRAM_FAILURE

<program source>:10:10: fatal error: 'inc_vendor.cl' file not found
#include "inc_vendor.cl"
         ^

* Device #3: Kernel /Users/me/GitHub/hashcat/OpenCL/m00900_a3-optimized.cl build failed - proceeding without this device.

Started: Wed Jul 26 12:51:50 2017
Stopped: Wed Jul 26 12:51:51 2017
pc-me:hashcat root# 
```

So, how can I fix this problem? Any idea?

**Comment by Hydraze** [comment]
I can't reproduce on macOS Sierra or El Capitan.
By the way, why are you running hashcat as root?

**Comment by ccakgunduz** [comment]
I didn't run root specifically for the hashcat. In parallel I needed to be root some other stuff and I just basically  didn't use the sudo for every command. Just run with root.

**Comment by ccakgunduz** [comment]
Interesting. I've just tried to run it with unprivileged mode and it works. @Hydraze thank you for your btw question, lol.

**Comment by ccakgunduz** [comment]
By the way I've check one of my friend who has upper version of MacBook Pro. So all device type in his MacBook is one level up than mine. He can successfully run hashcat as a root. So as I said, interesting.

**Comment by Hydraze** [comment]
I can also run hashcat as root on my mac so I think there is an issue with your installation rather than with hashcat itself

**Comment by ccakgunduz** [comment]
I use the github version of hashcat.  I just followed the instructions on [Build.md](https://github.com/hashcat/hashcat/blob/master/BUILD.md) neither more nor less. So I'm not sure that it is my problem or something else.

**Comment by ccakgunduz** [comment]
There is another funny thing. Yesterday I almost tried 10 times for running `./hashcat --benchmark` as a root and every time it gave `inc_vendor.cl file not found` error. After you said  

> why are you using hashcat as root?

, I ran hashcat as root first and got the same error. Then I ran it as a normal user and it worked perfectly. 

The funny stuff is I've just run the hashcat as root and there is no problem right now. It works with root user too.
